api/src/controllers/post/collection-bcmi.js

```diff
@@ -0,0 +1,84 @@
+const mongoose = require('mongoose');
```

Comment:
Do we need any explicit checks for a user having the `admin:bmi` role for these? Looks like from swagger it'll let through `admin:nrced` and `admin:lng` even though those admins shouldn't have access to post a collection?

Reply:
Yeah good call, Ill add a check

Reply:
I also was setting all admins with read/write access, so I've fixed that to just be sysadmin and bcmi